FWCore/Services/src/InitRootHandlers.cc

```diff
@@ -272,7 +271,7 @@ namespace edm {
 
       // Load the library containing dictionaries for std:: classes, if not already loaded.
       if (!hasDictionary(typeid(std::vector<std::vector<unsigned int> >))) {
-         edmplugin::PluginCapabilities::get()->load(dictionaryPlugInPrefix() + "std::vector<std::vector<unsigned int> >");
+         TypeWithDict::byName("std::vector<std::vector<unsigned int> >");
```

Comment:
I'm not sure.  What it does do is load libDataFormatsStdDectionaries.so if it is not already loaded.  If you want, I can remove this in another PR and see if anything breaks
I can't remember if we need that library loaded up front.